kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/Service2Service.java

```diff
@@ -59,7 +59,7 @@ public TaskExchange addToken(TaskExchange taskExchange) {
 			// We need to fetch the latest token for the sourceUser
 			log.debug(taskExchange.getSourceCode() + " : No token found, fetching latest token");
 			BaseEntity userBE = beUtils.getBaseEntity(taskExchange.getSourceCode());
-			BaseEntity project = beUtils.getBaseEntity(Prefix.PRJ.concat("_".concat(userBE.getRealm().toUpperCase())));
+			BaseEntity project = beUtils.getBaseEntity(Prefix.PRJ_.concat("_".concat(userBE.getRealm().toUpperCase())));
```

Comment:
Is this going to evaluate to `PRJ__<PRODUCTCODE>` instead of `PRJ_<PRODUCTCODE>`?

Reply:
Variable name is representative of value, so this would have been an issue, thanks.